muesli/web/viewsUser.py

```diff
@@ -21,28 +21,32 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from muesli import models
-from muesli.web.forms import *
-from muesli.web.context import *
+from muesli import utils
+from muesli.web import forms
+from muesli.web import context
+from muesli.types import Term
+import muesli
 
 from pyramid import security
 from pyramid.view import view_config
 from pyramid.response import Response
 from pyramid.httpexceptions import HTTPNotFound, HTTPBadRequest, HTTPFound, HTTPForbidden
 from pyramid.url import route_url
 from sqlalchemy.orm import exc
-from sqlalchemy import func, or_
+from sqlalchemy import func, or_, not_
 from hashlib import sha1
-
 from muesli.mail import Message, sendMail
 
 import re
 import os
 import datetime
 import collections
+import binascii
```

Comment:
you don't seem to use this library anywhere

Reply:
Yup. Please await https://github.com/Philipp-g/muesli/issues/89 for such changes.